deployment/bare_metal/.env.dist

```diff
@@ -38,7 +38,7 @@ KLICKTIPP_PASSWORD=
 KLICKTIPP_APIKEY_DE=
 KLICKTIPP_APIKEY_EN=
 
-EMAIL=false
+EMAIL=true
```

Comment:
`EMAIL=true`

Why is this not in the master.
It's not made by me.

May it came in by conflict solving on my side?

Reply:
has been settled